pyproject.toml

```
@@ -0,0 +1,19 @@
+[tool.poetry]
+name = "docker-socket-proxy"
+version = "0.1.0"
```

This is IMHO a better placeholder for things that aren't meant to be python packages.

```suggestion
version = "0.0.0"
```